auth/helper-scripts/get-cloudformation-outputs.sh

```diff
@@ -29,6 +29,9 @@ else
   echo "" > "$ENV_FILE_PATH"
 fi
 
+# --- Add a newline separator if .env.static was copied or file was just created ---
+echo "" >> "$ENV_FILE_PATH"
```

Comment:
Without newline it causes the cloudformation outsputs to be joined onto the same line as static env